packages/ses/README.md

```diff
@@ -503,6 +502,153 @@ system generated other diagnostic information hidden in side tables. The tamed
 console uses these side tables to output more informative diagnostics.
 [Logging Errors](./src/error/README.md) explains the design.
 
+## Security claims and caveats
+
+Provided that the `ses` implementation and its trusted compute base are
+correct, we claim that a host program (Figure 1) can evaluate a guest program
```

Comment:
Given the tc39 use of the term "host", if you want to use that here, you need to explain what you mean an do not mean early. Indeed, this text almost adequately explains it by example. But it doesn't impede the misunderstanding.

Reply:
Yeah I'm conflicted. While it's good to make the distinction with guest programs below, the host terminology is a bit overloaded.

Reply:
Adding a terminology sentence above.